plugins/Marketplace/tests/UI/Marketplace_spec.js

```diff
@@ -97,14 +97,15 @@ describe("Marketplace", function () {
 
         testEnvironment.consumer = consumer;
         testEnvironment.mockMarketplaceApiService = 1;
+        testEnvironment.forceEnablePluginUpdateChecks = forceEnableUpdateChecks ? 1 : 0;
         testEnvironment.save();
     }
 
     ['superuser', 'user', 'multiUserEnvironment'].forEach(function (mode) {
 
         if (mode !== 'user') {
             it('should show available updates in plugins page', async function() {
-                setEnvironment(mode, noLicense);
+                setEnvironment(mode, noLicense, true);
```

Comment:
It works together with the update checks in the backend that were changed in this PR as well, so it still correctly checks for updates here, but otherwise ignore them for all other UI tests.